react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -133,6 +138,11 @@ public void requestImageFailedRetryDialog(int mediaId) {
             public void requestImageUploadCancelDialog(int mediaId) {
                 mOnMediaLibraryButtonListener.onCancelUploadForMediaClicked(mediaId);
             }
+
+            @Override
+            public void editorDidMount(Boolean hasUnsupportedBlocks) {
```

Comment:
Let's replace `Boolean` with `boolean` here (it's a different class 😬) since the latter is what the interface is expecting.

Reply:
Fix in: eb066b6